
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://GenomicsStandardsConsortium.github.io/mixs-missing-value-sandbox/">
      
      <link rel="icon" href="assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-8.5.10">
    
    
      
        <title>mixs-missing-value-sandbox</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.975780f9.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.2505c338.min.css">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="None" data-md-color-accent="None">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#mixs-missing-value-sandbox" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="." title="mixs-missing-value-sandbox" class="md-header__button md-logo" aria-label="mixs-missing-value-sandbox" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            mixs-missing-value-sandbox
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Index
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/GenomicsStandardsConsortium/mixs-missing-value-sandbox" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="." title="mixs-missing-value-sandbox" class="md-nav__button md-logo" aria-label="mixs-missing-value-sandbox" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    mixs-missing-value-sandbox
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/GenomicsStandardsConsortium/mixs-missing-value-sandbox" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          Index
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="." class="md-nav__link md-nav__link--active">
        Index
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    Classes
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    Enumerations
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    Types
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    Subsets
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="about/" class="md-nav__link">
        About
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    Classes
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    Enumerations
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    Types
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    Subsets
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/GenomicsStandardsConsortium/mixs-missing-value-sandbox/edit/master/docs/index.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


<h1 id="mixs-missing-value-sandbox">mixs-missing-value-sandbox</h1>
<p>experimentation with formal representations of missing data</p>
<p>URI: https://w3id.org/GenomicsStandardsConsortium/mixs-missing-value-sandbox</p>
<p>Name: mixs-missing-value-sandbox</p>
<h2 id="classes">Classes</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Checklist/">Checklist</a></td>
<td>A collection of metadata terms (slots) to minimally describe the sampling and sequencing method of a specimen used to generate a nucleotide sequence.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Mims/">Mims</a></td>
<td>Metagenome or Environmental</td>
</tr>
<tr>
<td><a href="Extension/">Extension</a></td>
<td>A collection of recommended metadata terms (slots) developed by community experts, describing the specific context under which a sample was collected.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Soil/">Soil</a></td>
<td>A collection of terms appropriate when collecting samples and sequencing samples  obtained from the uppermost layer of Earth's crust, contributed by the Terragenome Consortium.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="MimsSoil/">MimsSoil</a></td>
<td>MIxS Data that comply with the Mims checklist and the Soil Extension</td>
</tr>
<tr>
<td><a href="MixsCompliantData/">MixsCompliantData</a></td>
<td>A collection of Data that comply with some combination of a MIxS checklist and environmental extension</td>
</tr>
</tbody>
</table>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="adapters/">adapters</a></td>
<td>Adapters provide priming sequences for both amplification and sequencing of t...</td>
</tr>
<tr>
<td><a href="agrochem_addition/">agrochem_addition</a></td>
<td>Addition of fertilizers, pesticides, etc</td>
</tr>
<tr>
<td><a href="al_sat/">al_sat</a></td>
<td>Aluminum saturation (esp</td>
</tr>
<tr>
<td><a href="al_sat_meth/">al_sat_meth</a></td>
<td>Reference or method used in determining Al saturation</td>
</tr>
<tr>
<td><a href="alt/">alt</a></td>
<td>Heights of objects such as airplanes, space shuttles, rockets, atmospheric ba...</td>
</tr>
<tr>
<td><a href="annot/">annot</a></td>
<td>Tool used for annotation, or for cases where annotation was provided by a com...</td>
</tr>
<tr>
<td><a href="annual_precpt/">annual_precpt</a></td>
<td>The average of all annual precipitation values known, or an estimated equival...</td>
</tr>
<tr>
<td><a href="annual_temp/">annual_temp</a></td>
<td>Mean annual temperature</td>
</tr>
<tr>
<td><a href="assembly_name/">assembly_name</a></td>
<td>Name/version of the assembly provided by the submitter that is used in the ge...</td>
</tr>
<tr>
<td><a href="assembly_qual/">assembly_qual</a></td>
<td>The assembly quality category is based on sets of criteria outlined for each ...</td>
</tr>
<tr>
<td><a href="assembly_software/">assembly_software</a></td>
<td>Tool(s) used for assembly, including version number and parameters</td>
</tr>
<tr>
<td><a href="associated_resource/">associated_resource</a></td>
<td>A related resource that is referenced, cited, or otherwise associated to the ...</td>
</tr>
<tr>
<td><a href="collection_date/">collection_date</a></td>
<td>The time of sampling, either as an instance (single point in time) or interva...</td>
</tr>
<tr>
<td><a href="crop_rotation/">crop_rotation</a></td>
<td>Whether or not crop is rotated, and if yes, rotation schedule</td>
</tr>
<tr>
<td><a href="cur_land_use/">cur_land_use</a></td>
<td>Present state of sample site</td>
</tr>
<tr>
<td><a href="cur_vegetation/">cur_vegetation</a></td>
<td>Vegetation classification from one or more standard classification systems, o...</td>
</tr>
<tr>
<td><a href="cur_vegetation_meth/">cur_vegetation_meth</a></td>
<td>Reference or method used in vegetation classification</td>
</tr>
<tr>
<td><a href="depth/">depth</a></td>
<td>The vertical distance below local surface</td>
</tr>
<tr>
<td><a href="drainage_class/">drainage_class</a></td>
<td>Drainage classification from a standard system such as the USDA system</td>
</tr>
<tr>
<td><a href="elev/">elev</a></td>
<td>Elevation of the sampling site is its height above a fixed reference point, m...</td>
</tr>
<tr>
<td><a href="env_broad_scale/">env_broad_scale</a></td>
<td>Report the major environmental system the sample or specimen came from</td>
</tr>
<tr>
<td><a href="env_local_scale/">env_local_scale</a></td>
<td>Report the entity or entities which are in the sample or specimen s local vic...</td>
</tr>
<tr>
<td><a href="env_medium/">env_medium</a></td>
<td>Report the environmental material(s) immediately surrounding the sample or sp...</td>
</tr>
<tr>
<td><a href="experimental_factor/">experimental_factor</a></td>
<td>Variable aspects of an experiment design that can be used to describe an expe...</td>
</tr>
<tr>
<td><a href="extreme_event/">extreme_event</a></td>
<td>Unusual physical events that may have affected microbial populations</td>
</tr>
<tr>
<td><a href="fao_class/">fao_class</a></td>
<td>Soil classification from the FAO World Reference Database for Soil Resources</td>
</tr>
<tr>
<td><a href="feat_pred/">feat_pred</a></td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc</td>
</tr>
<tr>
<td><a href="fire/">fire</a></td>
<td>Historical and/or physical evidence of fire</td>
</tr>
<tr>
<td><a href="flooding/">flooding</a></td>
<td>Historical and/or physical evidence of flooding</td>
</tr>
<tr>
<td><a href="geo_loc_name/">geo_loc_name</a></td>
<td>The geographical origin of the sample as defined by the country or sea name f...</td>
</tr>
<tr>
<td><a href="heavy_metals/">heavy_metals</a></td>
<td>Heavy metals present in the sequenced sample and their concentrations</td>
</tr>
<tr>
<td><a href="heavy_metals_meth/">heavy_metals_meth</a></td>
<td>Reference or method used in determining heavy metals</td>
</tr>
<tr>
<td><a href="horizon_meth/">horizon_meth</a></td>
<td>Reference or method used in determining the horizon</td>
</tr>
<tr>
<td><a href="lat_lon/">lat_lon</a></td>
<td>The geographical origin of the sample as defined by latitude and longitude</td>
</tr>
<tr>
<td><a href="lib_layout/">lib_layout</a></td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
</tr>
<tr>
<td><a href="lib_reads_seqd/">lib_reads_seqd</a></td>
<td>Total number of clones sequenced from the library</td>
</tr>
<tr>
<td><a href="lib_screen/">lib_screen</a></td>
<td>Specific enrichment or screening methods applied before and/or after creating...</td>
</tr>
<tr>
<td><a href="lib_size/">lib_size</a></td>
<td>Total number of clones in the library prepared for the project</td>
</tr>
<tr>
<td><a href="lib_vector/">lib_vector</a></td>
<td>Cloning vector type(s) used in construction of libraries</td>
</tr>
<tr>
<td><a href="link_addit_analys/">link_addit_analys</a></td>
<td>Link to additional analysis results performed on the sample</td>
</tr>
<tr>
<td><a href="link_class_info/">link_class_info</a></td>
<td>Link to digitized soil maps or other soil classification information</td>
</tr>
<tr>
<td><a href="link_climate_info/">link_climate_info</a></td>
<td>Link to climate resource</td>
</tr>
<tr>
<td><a href="local_class/">local_class</a></td>
<td>Soil classification based on local soil classification system</td>
</tr>
<tr>
<td><a href="local_class_meth/">local_class_meth</a></td>
<td>Reference or method used in determining the local soil classification</td>
</tr>
<tr>
<td><a href="micro_biomass_meth/">micro_biomass_meth</a></td>
<td>Reference or method used in determining microbial biomass</td>
</tr>
<tr>
<td><a href="microbial_biomass/">microbial_biomass</a></td>
<td>The part of the organic matter in the soil that constitutes living microorgan...</td>
</tr>
<tr>
<td><a href="mid/">mid</a></td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to spe...</td>
</tr>
<tr>
<td><a href="mims_soil_data/">mims_soil_data</a></td>
<td>Data that comply with Mims combined with Soil</td>
</tr>
<tr>
<td><a href="misc_param/">misc_param</a></td>
<td>Any other measurement performed or parameter collected, that is not listed he...</td>
</tr>
<tr>
<td><a href="neg_cont_type/">neg_cont_type</a></td>
<td>The substance or equipment used as a negative control in an investigation</td>
</tr>
<tr>
<td><a href="nucl_acid_amp/">nucl_acid_amp</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
</tr>
<tr>
<td><a href="nucl_acid_ext/">nucl_acid_ext</a></td>
<td>A link to a literature reference, electronic resource or a standard operating...</td>
</tr>
<tr>
<td><a href="number_contig/">number_contig</a></td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a giv...</td>
</tr>
<tr>
<td><a href="org_matter/">org_matter</a></td>
<td>Concentration of organic matter</td>
</tr>
<tr>
<td><a href="org_nitro/">org_nitro</a></td>
<td>Concentration of organic nitrogen</td>
</tr>
<tr>
<td><a href="ph/">ph</a></td>
<td>Ph measurement of the sample, or liquid portion of sample, or aqueous phase o...</td>
</tr>
<tr>
<td><a href="ph_meth/">ph_meth</a></td>
<td>Reference or method used in determining pH</td>
</tr>
<tr>
<td><a href="pool_dna_extracts/">pool_dna_extracts</a></td>
<td>Indicate whether multiple DNA extractions were mixed</td>
</tr>
<tr>
<td><a href="pos_cont_type/">pos_cont_type</a></td>
<td>The substance, mixture, product, or apparatus used to verify that a process w...</td>
</tr>
<tr>
<td><a href="prev_land_use_meth/">prev_land_use_meth</a></td>
<td>Reference or method used in determining previous land use and dates</td>
</tr>
<tr>
<td><a href="previous_land_use/">previous_land_use</a></td>
<td>Previous land use and dates</td>
</tr>
<tr>
<td><a href="profile_position/">profile_position</a></td>
<td>Cross-sectional position in the hillslope where sample was collected</td>
</tr>
<tr>
<td><a href="project_name/">project_name</a></td>
<td>Name of the project within which the sequencing was organized</td>
</tr>
<tr>
<td><a href="ref_biomaterial/">ref_biomaterial</a></td>
<td>Primary publication if isolated before genome publication; otherwise, primary...</td>
</tr>
<tr>
<td><a href="ref_db/">ref_db</a></td>
<td>List of database(s) used for ORF annotation, along with version number and re...</td>
</tr>
<tr>
<td><a href="rel_to_oxygen/">rel_to_oxygen</a></td>
<td>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic ...</td>
</tr>
<tr>
<td><a href="samp_collect_device/">samp_collect_device</a></td>
<td>The device used to collect an environmental sample</td>
</tr>
<tr>
<td><a href="samp_collect_method/">samp_collect_method</a></td>
<td>The method employed for collecting the sample</td>
</tr>
<tr>
<td><a href="samp_mat_process/">samp_mat_process</a></td>
<td>A brief description of any processing applied to the sample during or after r...</td>
</tr>
<tr>
<td><a href="samp_name/">samp_name</a></td>
<td>A local identifier or name that for the material sample used for extracting n...</td>
</tr>
<tr>
<td><a href="samp_size/">samp_size</a></td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample coll...</td>
</tr>
<tr>
<td><a href="samp_taxon_id/">samp_taxon_id</a></td>
<td>NCBI taxon id of the sample</td>
</tr>
<tr>
<td><a href="samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a></td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extractio...</td>
</tr>
<tr>
<td><a href="season_precpt/">season_precpt</a></td>
<td>The average of all seasonal precipitation values known, or an estimated equiv...</td>
</tr>
<tr>
<td><a href="season_temp/">season_temp</a></td>
<td>Mean seasonal temperature</td>
</tr>
<tr>
<td><a href="seq_meth/">seq_meth</a></td>
<td>Sequencing machine used</td>
</tr>
<tr>
<td><a href="sieving/">sieving</a></td>
<td>Collection design of pooled samples and/or sieve size and amount of sample si...</td>
</tr>
<tr>
<td><a href="sim_search_meth/">sim_search_meth</a></td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
</tr>
<tr>
<td><a href="size_frac/">size_frac</a></td>
<td>Filtering pore size used in sample preparation</td>
</tr>
<tr>
<td><a href="slope_aspect/">slope_aspect</a></td>
<td>The direction a slope faces</td>
</tr>
<tr>
<td><a href="slope_gradient/">slope_gradient</a></td>
<td>Commonly called 'slope'</td>
</tr>
<tr>
<td><a href="soil_data/">soil_data</a></td>
<td>Data that comply with Extension Soil</td>
</tr>
<tr>
<td><a href="soil_horizon/">soil_horizon</a></td>
<td>Specific layer in the land area which measures parallel to the soil surface a...</td>
</tr>
<tr>
<td><a href="soil_texture/">soil_texture</a></td>
<td>The relative proportion of different grain sizes of mineral particles in a so...</td>
</tr>
<tr>
<td><a href="soil_texture_meth/">soil_texture_meth</a></td>
<td>Reference or method used in determining soil texture</td>
</tr>
<tr>
<td><a href="soil_type/">soil_type</a></td>
<td>Description of the soil type or classification</td>
</tr>
<tr>
<td><a href="soil_type_meth/">soil_type_meth</a></td>
<td>Reference or method used in determining soil series name or other lower-level...</td>
</tr>
<tr>
<td><a href="sop/">sop</a></td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, ...</td>
</tr>
<tr>
<td><a href="source_mat_id/">source_mat_id</a></td>
<td>A unique identifier assigned to a material sample (as defined by http://rs</td>
</tr>
<tr>
<td><a href="store_cond/">store_cond</a></td>
<td>Explain how and for how long the soil sample was stored before DNA extraction...</td>
</tr>
<tr>
<td><a href="tax_class/">tax_class</a></td>
<td>Method used for taxonomic classification, along with reference database used,...</td>
</tr>
<tr>
<td><a href="temp/">temp</a></td>
<td>Temperature of the sample at the time of sampling</td>
</tr>
<tr>
<td><a href="tillage/">tillage</a></td>
<td>Note method(s) used for tilling</td>
</tr>
<tr>
<td><a href="tot_nitro_cont_meth/">tot_nitro_cont_meth</a></td>
<td>Reference or method used in determining the total nitrogen</td>
</tr>
<tr>
<td><a href="tot_nitro_content/">tot_nitro_content</a></td>
<td>Total nitrogen content of the sample</td>
</tr>
<tr>
<td><a href="tot_org_c_meth/">tot_org_c_meth</a></td>
<td>Reference or method used in determining total organic carbon</td>
</tr>
<tr>
<td><a href="tot_org_carb/">tot_org_carb</a></td>
<td>Total organic carbon content</td>
</tr>
<tr>
<td><a href="water_cont_soil_meth/">water_cont_soil_meth</a></td>
<td>Reference or method used in determining the water content of soil</td>
</tr>
<tr>
<td><a href="water_content/">water_content</a></td>
<td>Water content measurement</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="DrainageClassEnum/">DrainageClassEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="FaoClassEnum/">FaoClassEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="LibLayoutEnum/">LibLayoutEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="LithologyEnum/">LithologyEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="NegContTypeEnum/">NegContTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ProfilePositionEnum/">ProfilePositionEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="RelToOxygenEnum/">RelToOxygenEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SoilHorizonEnum/">SoilHorizonEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="TillageEnum/">TillageEnum</a></td>
<td></td>
</tr>
</tbody>
</table>
<h2 id="types">Types</h2>
<table>
<thead>
<tr>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Boolean/">Boolean</a></td>
<td>A binary (true or false) value</td>
</tr>
<tr>
<td><a href="Curie/">Curie</a></td>
<td>a compact URI</td>
</tr>
<tr>
<td><a href="Date/">Date</a></td>
<td>a date (year, month and day) in an idealized calendar</td>
</tr>
<tr>
<td><a href="DateOrDatetime/">DateOrDatetime</a></td>
<td>Either a date or a datetime</td>
</tr>
<tr>
<td><a href="Datetime/">Datetime</a></td>
<td>The combination of a date and time</td>
</tr>
<tr>
<td><a href="Decimal/">Decimal</a></td>
<td>A real number with arbitrary precision that conforms to the xsd:decimal speci...</td>
</tr>
<tr>
<td><a href="Double/">Double</a></td>
<td>A real number that conforms to the xsd:double specification</td>
</tr>
<tr>
<td><a href="Float/">Float</a></td>
<td>A real number that conforms to the xsd:float specification</td>
</tr>
<tr>
<td><a href="Integer/">Integer</a></td>
<td>An integer</td>
</tr>
<tr>
<td><a href="Jsonpath/">Jsonpath</a></td>
<td>A string encoding a JSON Path</td>
</tr>
<tr>
<td><a href="Jsonpointer/">Jsonpointer</a></td>
<td>A string encoding a JSON Pointer</td>
</tr>
<tr>
<td><a href="Ncname/">Ncname</a></td>
<td>Prefix part of CURIE</td>
</tr>
<tr>
<td><a href="Nodeidentifier/">Nodeidentifier</a></td>
<td>A URI, CURIE or BNODE that represents a node in a model</td>
</tr>
<tr>
<td><a href="Objectidentifier/">Objectidentifier</a></td>
<td>A URI or CURIE that represents an object in the model</td>
</tr>
<tr>
<td><a href="Sparqlpath/">Sparqlpath</a></td>
<td>A string encoding a SPARQL Property Path</td>
</tr>
<tr>
<td><a href="String/">String</a></td>
<td>A character string</td>
</tr>
<tr>
<td><a href="Time/">Time</a></td>
<td>A time object represents a (local) time of day, independent of any particular...</td>
</tr>
<tr>
<td><a href="Uri/">Uri</a></td>
<td>a complete URI</td>
</tr>
<tr>
<td><a href="Uriorcurie/">Uriorcurie</a></td>
<td>a URI or a CURIE</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="CombinationClasses/">CombinationClasses</a></td>
<td></td>
</tr>
<tr>
<td><a href="Environment/">Environment</a></td>
<td></td>
</tr>
<tr>
<td><a href="Investigation/">Investigation</a></td>
<td></td>
</tr>
<tr>
<td><a href="NucleicAcidSequenceSource/">NucleicAcidSequenceSource</a></td>
<td></td>
</tr>
<tr>
<td><a href="Sequencing/">Sequencing</a></td>
<td></td>
</tr>
</tbody>
</table>





                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
    <nav class="md-footer__inner md-grid" aria-label="Footer" >
      
      
        
        <a href="about/" class="md-footer__link md-footer__link--next" aria-label="Next: About" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              About
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": ".", "features": ["content.tabs.link"], "search": "assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="assets/javascripts/bundle.5a2dcb6a.min.js"></script>
      
    
    
  </body>
</html>